src/release_artifacts_resources/ios/cdk/cdk/credential_rotation/iam_construct.py

```diff
@@ -31,11 +31,11 @@ def create_circleci_release_proceess_role(self, bucket_arn: str, cloudfront_arn:
             role_name="CircleCIReleaseProcessRole",
             max_session_duration=core.Duration.hours(4),
         )
-
+        bucket_resource = bucket_arn + "/iosv2sdk/*"
```

Comment:
let's use a prefix of `"/aws-sdk-ios/*"` to keep it in line with the repo tag
```suggestion
        bucket_resource = bucket_arn + "/aws-sdk-ios/*"
```